src/main/java/dev/hephaestus/glowcase/packet/C2SEditEntityDisplayBlock.java

```diff
@@ -0,0 +1,69 @@
+package dev.hephaestus.glowcase.packet;
+
+import dev.hephaestus.glowcase.Glowcase;
+import dev.hephaestus.glowcase.block.entity.EntityDisplayBlockEntity;
+import dev.hephaestus.glowcase.block.entity.ItemDisplayBlockEntity;
+import net.minecraft.block.entity.BlockEntity;
+import net.minecraft.network.RegistryByteBuf;
+import net.minecraft.network.codec.PacketCodec;
+import net.minecraft.network.codec.PacketCodecs;
+import net.minecraft.network.packet.CustomPayload;
+import net.minecraft.server.world.ServerWorld;
+import net.minecraft.state.property.Properties;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.RotationPropertyHelper;
+
+public record C2SEditEntityDisplayBlock(BlockPos pos, ItemDisplayBlockEntity.RotationType rotationType, ItemDisplayBlockEntity.GivesItem givesItem, ItemDisplayBlockEntity.Offset offset, EntityDisplayBlockValues values) implements C2SEditBlockEntity {
```

Comment:
This should not have a `GivesItem` value

Reply:
Toggling giving the item (yes/once/no) is part of the entity display block, just like the item display block(which is what this packet was based on - the item display's packet).
Have I misunderstood something?

Reply:
I think chai was thinking this was only for display, but this actually hands out a spawn egg, right?

Reply:
Entity display blocks should not be interactable to adventure mode players, much less give out their spawn egg—use the item display block for that (the current item display block will be renamed to better reflect its behavior as an item provider)

Reply:
I was intending for it to give out the spawn eggs, but I will change it to not give out anything.